egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -160,9 +169,9 @@ def AddOutputLayer(config_lines, input, label_delay=None, suffix=None):
         name = '{0}-{1}'.format(name, suffix)
 
     if label_delay is None:
-        component_nodes.append('output-node name={0} input={1}'.format(name, input['descriptor']))
+        component_nodes.append('output-node name={0} input={1} objective={2}'.format(name, input['descriptor'], objective_type))
     else:
-        component_nodes.append('output-node name={0} input=Offset({1},{2})'.format(name, input['descriptor'], label_delay))
+        component_nodes.append('output-node name={0} input=Offset({1},{2}) objective={3}'.format(name, input['descriptor'], label_delay, objective_type))
```

Comment:
The rest of components.py does not have named arguments. Should I modify all of it?